Python/Product/Debugger/Debugger/DebugEngine/AD7BoundBreakpoint.cs

```diff
@@ -51,7 +50,7 @@ class AD7BoundBreakpoint : IDebugBoundBreakpoint2 {
 
             if (!_deleted) {
                 _deleted = true;
-                _breakpoint.Remove();
+                TaskExtensions.RunSynchronouslyOnUIThread(ct => _breakpoint.RemoveAsync(ct));
```

Comment:
If not, making it a method of some object that we always have here (`_engine` maybe?) would work (and might also allow us to detect timeouts more reliably and send telemetry info if we detect problems)